test/CppTests/TestConfigurations/config_lldb.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<TestMachineConfiguration>
+  <TestConfigurations>
+    <TestConfiguration
+      Compiler="Clang"
+      DebuggeeArchitecture="x64"
+      Debugger="Lldb" />
+  </TestConfigurations>
+  <Compilers>
+    <Compiler
+      Name="Clang"
+      Type="ClangPlusPlus"
+      Path="/usr/bin/clang++" />
+  </Compilers>
+  <Debuggers>
+    <!-- By not specifying the Path, LLDB that ships with the extension will be used. -->
+    <Debugger
+      Name="Lldb"
+      Type="Lldb"
+      MIMode="lldb"
+      Path="/Users/runner/work/MIEngine/MIEngine/debugAdapters/lldb-mi/bin/lldb-mi" />
```

Comment:
>Path="/Users/runner/work/MIEngine/MIEngine/debugAdapters/lldb-mi/bin/lldb-mi" /> [](http://example.com/codeflow?start=6&length=80)

Would it make more sense to create a sym link in pull-request.yml and that way it would be easier to set things up if you wanted to run the tests on your box?

Reply:
What would the path point to or do you suggest setting up `lldb-mi` in `PATH`?

Reply:
My thought was to drop a symlink in the same way the the C++ extension expects (next to OpenDebugAD7???).

Reply:
Got it. Utilize https://github.com/microsoft/MIEngine/blob/8456c49b4731971b885c2909f9feaac4bdcd494d/src/OpenDebugAD7/MILaunchOptions.cs#L27-L50